server/routes.js

```diff
@@ -0,0 +1,19 @@
+var Database = require ('/database/config.js');
+
+module.exports = function(app){
+  // Add new user account to database
+  app.post('/addnewuser', function(req, res){
+    Database.Users.create ({
```

Comment:
create will create any username, so let's check if it already exists before creating it